esmvaltool/diag_scripts/impact/quick_insights.py

```diff
@@ -0,0 +1,210 @@
+"""Python example diagnostic."""
+import logging
+from datetime import datetime
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+import xarray as xr
+
+from esmvaltool.diag_scripts.shared import (
+    ProvenanceLogger,
+    get_diagnostic_filename,
+    get_plot_filename,
+    group_metadata,
+    run_diagnostic,
+    select_metadata,
+)
+
+logger = logging.getLogger(Path(__file__).stem)
+
+
+def log_provenance(filename, ancestors, caption, cfg):
+    # We'll add some extra code to make sure the output is saved and tracked
+    provenance = {
+        'caption': caption,
+        'domains': ['reg'],
+        'authors': ['kalverla_peter'],
+        'references': ['isenes3'],
```

Comment:
Should this be projects?

```suggestion
        'projects': ['isenes3'],
```